packages/sil-governance/src/components/BlockchainTime/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { makeStyles } from "@material-ui/styles";
+import { Block, Typography } from "medulas-react-components";
+import React from "react";
+import { ReadonlyDate } from "readonly-date";
+
+interface Props {
+  readonly lastBlockHeight: number;
+  readonly lastBlockTime: ReadonlyDate;
+}
+
+const useStyles = makeStyles({
+  container: {
+    textAlign: "center",
```

Comment:
Have you tried `align` prop of `Typography` component? https://material-ui.com/ru/api/typography/

Reply:
No, thanks. That would work as well I guess. My idea is to set the textAlign for the whole container to include all `<Typography>`s